### ISSUE TITLE
Possibility of a release for people wanting to update to be able to migrate their templates to ASYNC?

### ISSUE THREAD

**Issue by DanielDornhardt** [meteor/blaze#441]
Hi Guys,

I know this isn't really a bug. But I wanted to ask:

If I see correctly there hasn't been a release since v2.7.1 from last May, right?

By now important changes have been done to support ASYNC data sources in templates. `#let`, `#if` / `#unless`, and even attributes now allow for reactive data sources.

These changes are almost required for everybody wanting to migrate their existing Blaze app to Async.

I think it'd be good to create a release including these changes in the near future so people wanting to migrate can use the package instead of manually checking out the blaze repositories locally.

What you you think?

**Comment by jankapunkt** [comment]
@DanielDornhardt releases of the packages are only possible by the Meteor Team // cc @Grubba27 @denihs

**Comment by Grubba27** [comment]
Yeah, that is true! Normally, I release when all is good(code, docs, changelog, and maybe write a blog post about what changed), and @jankapunkt or @StorytellerCZ ping me to do the release if you guys already did ask me to publish it. I'm sorry maybe it got lost in my to-do list. I thought it had some issues/PRs before it was considered ready to be published

**Comment by jankapunkt** [comment]
I actually don't know anymore. Maybe we didn't ping you actually....

**Comment by Grubba27** [comment]
That is okay. Maybe we should create a blaze v2.8 PR updating the changelog and packages, and I release it
